base-notebook/Dockerfile

```diff
@@ -5,6 +5,23 @@
 # https://hub.docker.com/_/ubuntu/?tab=tags&name=focal
 ARG ROOT_CONTAINER=ubuntu:focal
 
+## BEGIN MICROMAMBA WORKAROUND
+# For some reason, downloading micromamba with wget or curl leads to a segfault.
+# Download them in a separate stage with alpine.
+# Context: <https://github.com/jupyter/docker-stacks/pull/1594#issuecomment-1028426999>
+FROM alpine as download_micromamba_stage
+WORKDIR /tmp
+RUN set -x && \
+    arch=$(uname -m) && \
+    if [ "${arch}" = "x86_64" ]; then \
+        # Should be simpler, see <https://github.com/mamba-org/mamba/issues/1437>
+        arch="64"; \
+    fi && \
+    echo $arch && \
+    apk add --no-cache curl==7.80.0-r0 && \
```

Comment:
Could we not specify curl version here?

Reply:
Thanks for the review! I only did it to prevent hadolint from complaining. I also think it's weird; I'll add an ignore instead.

Reply:
Thanks! I thought we explicitly asked hadolint to ignore this rule be default in our project.